### ISSUE TITLE
File Permissions

### ISSUE THREAD

**Issue by jcaip1** [robertdevore/site-vitals-for-wordpress#4]
```
Result: Needs Attention
Recommendation: Insecure file permissions detected for: uploads. Set permissions to 644 for enhanced security.
```

I don't think that is suppose to be the case?  Isn't uploads suppose to be 755?

**Comment by robertdevore** [comment]
@jcaip1 yup, you're right. This one looks like it snuck through and needs to have the code tweaked for this one. I'll be pushing that out in an update soon 🙏

**Comment by jcaip1** [comment]
> @jcaip1 yup, you're right. This one looks like it snuck through and needs to have the code tweaked for this one. I'll be pushing that out in an update soon 🙏

Awesome, take your time.. I'm glad you put this plugin out there, this issue is more of an FYI.

:)

Thanks.
